app/src/main/java/com/jerboa/ui/components/common/AppBars.kt

```diff
@@ -112,99 +111,87 @@ fun SimpleTopAppBarPreview() {
     }
 }
 
-@Composable
-fun BottomAppBarAll(
+fun NavigationSuiteScope.adaptiveNavigationBar(
     selectedTab: NavTab,
-    onSelect: (NavTab) -> Unit,
+    onSelectTab: (NavTab) -> Unit,
     userViewType: UserViewType,
     unreadCounts: Long,
     unreadAppCount: Long?,
     unreadReportCount: Long?,
     showTextDescriptionsInNavbar: Boolean,
 ) {
-    // Check for preview mode
-    if (LocalContext.current is Activity) {
-        val window = (LocalContext.current as Activity).window
-        val colorScheme = MaterialTheme.colorScheme
-
-        DisposableEffect(Unit) {
```

Comment:
I see this is also gone but this was really necessary for the navigation bar to look the same as system navigation bar.

It looked quite odd. I'll check it out later to see how this looks now.

Reply:
I don't see any need for these now in the jetpack compose code, maybe they were necessary once, but now with `edgeToedge` , it seems to handle the navbar coloring fine.

Reply:
Not on older API levels